cwms-http-client/build.gradle

```diff
@@ -14,4 +14,22 @@ dependencies {
     testFixturesImplementation(platform("com.squareup.okhttp3:okhttp-bom:4.9.2"))
     testFixturesImplementation('com.squareup.okhttp3:okhttp:4.9.2')
     testFixturesImplementation('com.squareup.okhttp3:mockwebserver:4.9.2')
-}
\ No newline at end of file
+}
+
+publishing {
+    repositories {
+        maven {
+            url = version.endsWith("SNAPSHOT") ? System.getenv("DEPLOY_URL") + "/github-snapshots" : System.getenv("DEPLOY_URL") + "/github-releases"
```

Comment:
fyi, you could do a `def baseUrl = System.getenv("DEPLOY_URL")` either between lines 17 and 19 or 19 and 20 to reduce the duplication here.

Reply:
Also instead of duplicating this identical block 3 times, in the root build.gradle you can do 
```
subprojects {
   publishing {
     ...
   }
}
```

Reply:
The only item you would have in each sub module is the "publications" as that is actually different for each.

Reply:
good point, moved to the root build.gradle